tests/EventFactoryTest.php

```diff
@@ -16,20 +16,29 @@
 
 class EventFactoryTest extends TestCase
 {
+    /**
+     * @var Options
+     */
+    private $options;
+
+    protected function setUp(): void
+    {
+        $this->options = new Options();
```

Comment:
I don't see a real value in having this instance set as variable of the class, is there any reason you moved it here?

Reply:
Mostly for convenience, having `new Options()` in almost every test doesn't really make sense. Also, we've done this in other test classes.

Reply:
Not that it changes much, but only two tests are using the options so I don't think we should move the instantiation to the `setUp` method which creates an additional object each time a test case is executed even if there is no need for it